runtime/oti/j9nonbuilder.h

```diff
@@ -5538,6 +5542,19 @@ typedef struct J9JavaVM {
 	U_32 javaVM31;
 	U_32 javaVM31PadTo8; /* Possible to optimize with future guarded U_32 member in ENV_DATA64. */
 #endif /* defined(J9VM_ZOS_3164_INTEROPERABILITY) */
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	jclass criuResultTypeClass;
+	jfieldID criuSupportSuccess;
+	jfieldID criuSupportUnsupportedOperation;
+	jfieldID criuSupportInvalidArguments;
+	jfieldID criuSupportSystemCheckpointFailure;
+	jfieldID criuSupportJVMCheckpointFailure;
+	jfieldID criuSupportJVMRestoreFailure;
+	jclass criuResultClass;
+	jfieldID criuSupportSystemErrorCode;
+	jfieldID criuSupportThrowable;
+	jfieldID criuSupportType;
+#endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
```

Comment:
Adding entries to `vmconstantpool.xml` adds a startup cost. At some point, this feature will be enabled by default so that would be adding a cost to applications that have no use for it. Also, checkpointing the JVM is a slow path, so I'm not sure there is much to gain with `vmconstantpool.xml`.